### PR TITLE
changed script and stylesheet imports to cdn links and deleted static files

### PR DESCRIPTION
**Pull request by xit4**
I tried to keep versions the same as they were in the local files -> I did not upgrade the libraries to the latest versions

related to #19

### CODE REVIEW

**Comment by ayan-b** [comment]
Why so many changes?!

**Comment by xit4** [comment]
argh, it's the linter!
basically "fixed" indentation for the file

**Comment by ayan-b** [comment]
Please don't fix linting issues in this PR. If you wish, you can create another PR for linting issue fixes.

**Comment by xit4** [comment]
it makes sense, I'll revert the linting on this PR

**Comment by xit4** [comment]
@ayan-b I undid the indentation changes so we are left with just the CDN links and the removal of the static libraries

File: index.html [hunk 1]
```diff
@@ -11,13 +11,13 @@
       <!--Import Google Icon Font-->
       <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
       <!--Import materialize.css-->
-      <link type="text/css" rel="stylesheet" href="./static/materialize/css/materialize.min.css"  media="screen,projection"/>
+      <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```
Comment (ayan-b) [inline index.html]: Hi @xit4, nice job there! But can you please explain why did you remove link type and media? :smile:

Comment (xit4) [reply index.html]: I used the "suggested" <link> from the materialize [getting started page](https://materializecss.com/getting-started.html)

Anyway, this is how it would look like with those attributes.
```suggestion
      <link rel="stylesheet" type="text/css" media="screen,projection" href="https://cdnjs.cloudflare.com/ajax/libs/materialize/1.0.0/css/materialize.min.css">
```

Comment (ayan-b) [reply index.html]: Ok, no problem!
